ng-dev/pr/common/validation/assert-enforce-tested.ts

```diff
@@ -78,11 +78,11 @@ export async function pullRequestHasValidTestedComment(
   gitClient: AuthenticatedGitClient,
 ): Promise<boolean> {
   for (const {bodyText, author} of comments) {
-    if (
-      bodyText.startsWith(`TESTED=`) &&
-      (await githubMacros.isGooglerOrgMember(gitClient.github, author.login))
-    ) {
-      return true;
+    if (bodyText.startsWith(`TESTED=`)) {
+      try {
```

Comment:
Can we do this try catch work in the macro itself? Because this issue would exist for anything that relies on the macro

Reply:
Sure